agent-manager/Containerfile

```diff
@@ -0,0 +1,13 @@
+FROM registry.access.redhat.com/ubi9/python-312:latest
+
+USER root
+RUN dnf update -y && dnf clean all
+
+RUN pip3 install --no-cache-dir uv
+
+# Copy the project into the image
+COPY . /app
+
+# Install the application dependencies.
+WORKDIR /app
+RUN uv sync --frozen --no-cache --all-packages
```

Comment:
There does not seem to be a CMD ?

I was expecting that we could run this container and it would register the agents that were defined?